nodejs.org/s2i/bin/save-artifacts

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+pushd ${HOME} >/dev/null
```

Comment:
The example script you linked to is changing to the `$HOME` directory here so that the dependency path is relative - i.e. `deps`. You are using an absolute path below, so I don't think you need this. If you remove it, you won't need `popd` below.